src/Compilers/CSharp/Test/Symbol/Symbols/Source/FileModifierTests.cs

```diff
@@ -813,6 +818,74 @@ void symbolValidator(ModuleSymbol module)
         }
     }
 
+    private void DeterminismCore(
+        string fileTypeName,
+        CSharpCompilation comp1,
+        CSharpCompilation comp2,
+        CSharpCompilation comp3)
```

Comment:
>comp3 [](http://example.com/codeflow?start=26&length=5)

Is it necessary to pass in 3 compilations, or could we simply compare two compilations and have callers call this method twice?
```
DeterminismCore(name, comp1, comp2);
DeterminismCore(name, comp1, comp3);
```